ironfish/src/primitives/transaction.test.slow.ts

```diff
@@ -0,0 +1,29 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { createNodeTest, useAccountFixture } from '../testUtilities'
+
+describe('Accounts', () => {
+  const nodeTest = createNodeTest()
+
+  it('produces unique transaction hashes', async () => {
+    const account = await useAccountFixture(nodeTest.accounts)
+
+    const transactionA = await nodeTest.strategy.createMinersFee(
+      BigInt(0),
+      1,
+      account.spendingKey,
+    )
+
+    const transactionB = await nodeTest.strategy.createMinersFee(
+      BigInt(0),
+      1,
+      account.spendingKey,
+    )
+
+    const hashA = transactionA.transactionHash()
+    const hashB = transactionB.transactionHash()
+
+    expect(hashA.equals(hashB)).toBe(false)
```

Comment:
Could we do 
```ts
expect(hashA).not.toBe(hashB);
```

Reply:
toBe uses reference equalilty, so if you want to check that the buffers contents are actually equal or not we need to use Buffer.equals to check that the byte sets are the same or not.